frontend/benefit/handler/Dockerfile

```diff
@@ -0,0 +1,104 @@
+# =======================================
```

Comment:
dockerfile is also 95% copy-paste for each project. Perhaps we could make a jira ticket to use template and give the folder `benefit/handler` and some others as argument. Something like this: https://blog.dockbit.com/templating-your-dockerfile-like-a-boss-2a84a67d28e9

Reply:
yea, let create a ticket